components/03-modules/price-box/Price-box.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
For element I wouldn't use extra wrapper as not always it can be required and then we increase dom elements in end product or need to overwrite it. Of course we can also skip this component and just add wrapper in places where it will be required.